portal-kernel/src/com/liferay/portal/kernel/workflow/BaseWorkflowNode.java

```diff
@@ -26,15 +27,21 @@ public String getLabel(Locale locale) {
 			return HtmlUtil.escape(label);
 		}
 
+		Language language = LanguageUtil.getLanguage();
+
+		label = language.get(locale, _name);
```

Comment:
Hey @p-albuquerque sorry for my confusion, please use null as default `language.get(locale, _name, null); `and check if label if not null, this approach is more clear

Reply:
got it